lib/webpacker/configuration.rb

```diff
@@ -47,10 +47,10 @@ def fetch(key)
     end
 
     def data
-      if env.development?
-        refresh
-      else
+      if env.production?
```

Comment:
Rather than checking the env directly, we should have a configuration option that can be set. Maybe something like `cache_config` and `cache_manifest`. Then people can specify what they want in all sorts of different envs, like staging or beta or whatever.